src/main/java/seedu/address/model/person/Subject.java

```diff
@@ -115,7 +115,7 @@ public static boolean isValidSubjectsByLevel(Level level, Set<Subject> subjects)
         }
         EnumSet<Subjects> validSubjects = validSubjectsByLevel.get(level);
         for (Subject s: subjects) {
-            if (!validSubjects.contains(Subjects.valueOf(s.subjectName.toUpperCase()))) {
+            if (!validSubjects.contains(Subjects.valueOf(s.subjectName))) {
```

Comment:
same as above should still include .toUpperCase ?

Reply:
no need cos it is turned in subjects so its already toUpper()